tests/vmi_configuration_test.go

```diff
@@ -1674,13 +1674,14 @@ var _ = Describe("[sig-compute]Configurations", func() {
 
 			BeforeEach(func() {
 				By("Creating a runtime class")
-				createRuntimeClass(runtimeClassName, "custom-handler")
+				_, err := createRuntimeClass(runtimeClassName, "custom-handler")
+				Expect(err).NotTo(HaveOccurred(), "cannot create runtime-class "+runtimeClassName)
 			})
 
 			AfterEach(func() {
 				By("Cleaning up runtime class")
-				err = deleteRuntimeClass(runtimeClassName)
-				Expect(err).NotTo(HaveOccurred())
+				err := deleteRuntimeClass(runtimeClassName)
+				Expect(err).NotTo(HaveOccurred(), "cannot delete runtime-class "+runtimeClassName)
```

Comment:
nit: this could be inlined, right ? That would allow you to use positive logic - i.e. `Expect(deleteRuntimeClass(runtimeClassName)).To(Succeed())`

Reply:
Actually I disagree :)
I prefer to keep checks and actions separated, IMHO it's clearer that way

Reply:
fair enough.

Reply:
Right, I can see your point and it's a valid one.
At least in my eyes, tests are performing actions then asserting that the results are valid. When it's being done together at least I get confused and it's easier to miss test actions this way. Another problem is functions with multiple return values, then it might not be possible at all.

But it's a matter of taste and style I guess :)

Reply:
I honestly prefer to not have these assertion messages and also prefer the succeed. It is true that sometimes, a clearer fail message could be beneficial, but there is nothing more confusing than when these messages get outdated and cause confusion when debugging tests. Also very often they are not formulated accurately.

Especially in this case, there is a `deleteRuntimeClass` function checked if it succeeds, the "cannot delete runtime-class" is not really adding any value which I would not have guessed right away when I see `Exect(deleteRuntimeClass).To(Succeed())`.